ui-ngx/src/app/shared/models/public-api.ts

```diff
@@ -43,6 +43,7 @@ export * from './notification.models';
 export * from './websocket/notification-ws.models';
 export * from './websocket/websocket.models';
 export * from './oauth2.models';
+export * from './query/query.models
```

Comment:
This code creates the error:
TS2308: Module  './page/public-api'  has already exported a member named  Direction. Consider explicitly re-exporting to resolve the ambiguity.

Reply:
perhaps rename one of the Direction members?